csm_web/scheduler/views.py

```diff
@@ -298,3 +302,30 @@ class UserViewSet(*viewset_with('list')):
 
     def list(self, request):
         return Response(self.queryset.values_list('email', flat=True))
+
+class MentorBioInfoDetail(APIView):
+    renderer_classes = [TemplateHTMLRenderer]
+    parser_classes = [MultiPartParser]
+    template_name = 'mentor_bio_form.html'
+
+    # TODO:
+    # Find some way to restrict user field idiomatically
+    # Display photo in the form if it already exists
+    def get(self, request, format=None):
+        try:
+            info = MentorBioInfo.objects.get(user=self.request.user)
+        except MentorBioInfo.DoesNotExist:
+            info = MentorBioInfo()
+        serializer = MentorBioInfoSerializer(info)
+        return Response({'serializer': serializer})
+
+    def post(self, request, format=None):
+        try:
```

Comment:
Use `MentorBioInfo.objects.get_or_create` ([relevant Django docs](https://docs.djangoproject.com/en/3.0/ref/models/querysets/#get-or-create)) instead of all this boilerplate.